ChunkyDocs/docs/faq/index.md

```diff
@@ -1,62 +0,0 @@
-# Frequently Asked Questions
-
-Any question not answered here please ask on either our [Discord server](https://discord.gg/VqcHpsF) or on our [Reddit community](https://www.reddit.com/r/chunky/).
-
----
-
-## Why is there noise/grain/random bright dots in the render?
-
-This is not a bug, but an unfortunate effect of the rendering algorithm used in Chunky. Torches and other small light sources cause a very random illumination and it takes a long time to render such light nicely. For more information please read the **Path Tracing article**. You can disable emitters under the Lighting tab in the Render Controls to remove most of the random bright dots. Other sources of light are typically larger and clear up quickly though HDRi skymaps can be an issue. **Note that rendering for a longer time will eventually clear up the noise**, though it may take a very long time.
-  
-There are techniques and plugins which can help reduce noise. For more information please visit [jackjt8's Guide to Chunky - Denoising](https://jackjt8.github.io/ChunkyGuide/docs/advanced_techniques/denoising.html).
-
-
-## How long does it take to render an image?
-
-There is no exact answer to this question. The main thing that affects render time is your CPU, the size of the image, and the lighting conditions in the scene you are rendering. It can take anywhere from an hour to a couple of days to render a nice image. You can reduce the size of the canvas, disable emitters, enable Emitter Sampling, or use a denoising technique to speed up the convergence rate. See the **Path Tracing article** or [jackjt8's Guide to Chunky - Denoising](https://jackjt8.github.io/ChunkyGuide/docs/advanced_techniques/denoising.html) for more details.
-
-
-## Is GPU rendering supported?
-
-GPU rendering support is currently in development in the form of an OpenCL 1.2 renderer plugin. This renderer is still under development and many features of the CPU renderer are not yet supported. For more information check out the [OpenCL plugin GitHub](https://github.com/ThatRedox/ChunkyClPlugin).
```

Comment:
"GPU support is on hold for now" - Redox

Adjust wording to reflect that while there is a plugin it is not in active development.